tests/test_actors.py

```diff
@@ -606,3 +606,10 @@ def raise_rate_limit_exceeded():
         # Then warning mock should be called with a special message
         warning_messages = [args[0] for _, args, _ in warning_mock.mock_calls]
         assert "Rate limit exceeded in message %s: %s." in warning_messages
+
+
+def test_as_dict_actor(stub_broker, do_work):
+    res = do_work.as_dict()
+    assert res["name"] == "do_work"
```

Comment:
you can directly test `assert res === {...}`, it's better because if you add a new key you will see it.